docassemble/AssemblyLine/al_document.py

```diff
@@ -1448,7 +1448,7 @@ def _update_labels(self, auto_labeler: Callable = None) -> None:
             auto_labeler = self.auto_labeler
 
         for index, exhibit in enumerate(self.elements):
-            exhibit.label = self.auto_labeler(index)
+            exhibit.label = auto_labeler(index)
```

Comment:
I don't think this change is needed? You don't pass anything to the `auto_labeler` parameter on [link 1495](https://github.com/SuffolkLITLab/docassemble-AssemblyLine/pull/446/files#diff-db23667d014599821c0250a04d5ffbbcf327b89266e017c6f6304c30a19ff8efR1495), which means this will error our trying to call the function of `None`.

From the rest of the PR, it looks like you should just remove that parameter, and only use `self.auto_labeler`?

Reply:
Maybe YAGNI applies here. My thought was to give people an option to manually call `_update_labels()` in some circumstance I haven't foreseen.

I'm not sure where it would error out though? On line 1448+1449, I was previously checking for a `None` value of `auto_labeler` and if it was null, `auto_labeler` is set to `self.auto_labeler`.

Reply:
Ah, gotcha, my eyes skipped over the parts of the function not in this PR. That's fine, IMO. Testing now.